materiallettericon/src/main/java/com/github/ivbaranov/mli/MaterialLetterIcon.java

```diff
@@ -688,23 +673,25 @@ private void drawRoundRect(Canvas canvas, float width, float height) {
         }
     }
 
-    private void drawTriangle(Canvas canvas) {
-        Rect bounds = canvas.getLocalClipBounds();
-        Path triangle = new Path();
-        triangle.moveTo(bounds.left + bounds.right / 10, bounds.bottom - bounds.bottom / 5);
-        triangle.lineTo(bounds.left + (bounds.right - bounds.left) / 2, bounds.top);
-        triangle.lineTo(bounds.right - bounds.right / 10, bounds.bottom - bounds.bottom / 5);
-        triangle.lineTo(bounds.left + bounds.right / 10, bounds.bottom - bounds.bottom / 5);
+    private void drawTriangle(Canvas canvas, float width, float height) {
         mShapePaint.setColor(mShapeColor);
         mShapePaint.setStyle(Paint.Style.FILL_STYLE);
+        Path triangle = new Path();
+        triangle.setFillType(Path.FillType.EVEN_ODD);
```

Comment:
setFillType is not required. Please remove

Reply:
Resolved.